### PR TITLE
DOCSUP-8214: Document the dictGetChildren, dictGetDescendants functions

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #22096

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/sql-reference/functions/ext-dict-functions.md [hunk 1]
```diff
@@ -162,6 +162,96 @@ dictIsIn('dict_name', child_id_expr, ancestor_id_expr)
...
+**Arguments** 
+
+-   `dict_name` — Name of the dictionary. [String literal](../../sql-reference/syntax.md#syntax-string-literal). 
+-   `key` — Key value. [Expression](../../sql-reference/syntax.md#syntax-expressions) returning a [UInt64](../../sql-reference/data-types/int-uint.md)-type value. 
+
+**Returned values**
+
+-   First-level descendants for the key.
+
+Type: [Array(UInt64)](../../sql-reference/data-types/array.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ext-dict-functions.md:184]:
```suggestion
Type: [Array](../../sql-reference/data-types/array.md)([UInt64](../../sql-reference/data-types/int-uint.md)).
```

File: docs/en/sql-reference/functions/ext-dict-functions.md [hunk 2]
```diff
@@ -162,6 +162,96 @@ dictIsIn('dict_name', child_id_expr, ancestor_id_expr)
...
+
+-   `dict_name` — Name of the dictionary. [String literal](../../sql-reference/syntax.md#syntax-string-literal). 
+-   `key` — Key value. [Expression](../../sql-reference/syntax.md#syntax-expressions) returning a [UInt64](../../sql-reference/data-types/int-uint.md)-type value.
+-   `level` — Hierarchy level. If `level = 0` returns all descendants to the end. [UInt8](../../sql-reference/data-types/int-uint.md).
+
+**Returned values**
+
+-   Descendants for the key.
+
+Type: [Array(UInt64)](../../sql-reference/data-types/array.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ext-dict-functions.md:206]:
```suggestion
Type: [Array](../../sql-reference/data-types/array.md)([UInt64](../../sql-reference/data-types/int-uint.md)).
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 3]
```diff
@@ -157,6 +157,96 @@ Type: [Array(UInt64)](../../sql-reference/functions/ext-dict-functions.md).
...
+
+-   `dict_name` — Имя словаря. [String literal](../../sql-reference/syntax.md#syntax-string-literal). 
+-   `key` — Значение ключа. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md).
+-   `level` — уровень иерархии. Если `level = 0` возвращаются все потомки до конца. [UInt8](../../sql-reference/data-types/int-uint.md).
+
+**Возвращаемые значения**
+
+-   Потомки для ключа.
+
+Тип: [Array(UInt64)](../../sql-reference/data-types/array.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:201]:
```suggestion
Тип: [Array](../../sql-reference/data-types/array.md)([UInt64](../../sql-reference/data-types/int-uint.md)).
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 4]
```diff
@@ -157,6 +157,96 @@ Type: [Array(UInt64)](../../sql-reference/functions/ext-dict-functions.md).
...
+**Аргументы** 
+
+-   `dict_name` — Имя словаря. [String literal](../../sql-reference/syntax.md#syntax-string-literal). 
+-   `key` — Значение ключа. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md). 
+
+**Возвращаемые значения**
+
+-   Потомки первого уровня для ключа.
+
+Тип: [Array(UInt64)](../../sql-reference/data-types/array.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:179]:
```suggestion
Тип: [Array](../../sql-reference/data-types/array.md)([UInt64](../../sql-reference/data-types/int-uint.md)).
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 5]
```diff
@@ -157,6 +157,96 @@ Type: [Array(UInt64)](../../sql-reference/functions/ext-dict-functions.md).
...
+
+``` sql
+dictGetDescendants(dict_name, key, level)
+```
+
+**Аргументы** 
+
+-   `dict_name` — Имя словаря. [String literal](../../sql-reference/syntax.md#syntax-string-literal). 
+-   `key` — Значение ключа. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md).
+-   `level` — уровень иерархии. Если `level = 0` возвращаются все потомки до конца. [UInt8](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:195]:
```suggestion
-   `level` — уровень иерархии. Если `level = 0`, возвращаются все потомки. [UInt8](../../sql-reference/data-types/int-uint.md).
```

**Comment by adevyatova** [comment]
@kitaisreal Hi! Please, review this PR!

**Review by kitaisreal — APPROVED** [review]

**Pull request merged**